src/prettier/index.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { Tree } from '@angular-devkit/schematics';
 import {
   SchematicTestRunner,
-  UnitTestTree
+  UnitTestTree,
```

Comment:
Seems that trailing commas are applied by your editor's prettier configuration.

- [ ] Can you add the missing `.prettierrc` to our project, please? 😆 
- [ ] After this you can format this file and the trailing commas should be deleted.

You will find it in `src/prettier/templates/.prettierrc`.